esp/public/media/scripts/onsite/ajax_status.js

```diff
@@ -914,7 +914,7 @@ function render_table(display_mode, student_id)
                     new_td.append($j("<span/>").addClass("room").html(section.rooms));
                 }
                 //  TODO: make this snap to the right reliably
-                new_td.append($j("<span/>").addClass("studentcounts").attr("id", "studentcounts_" + section.id).html(section.num_students_checked_in.toString() + "/" + section.num_students_enrolled + "/" + section.capacity));
+                new_td.append($j("<span/>").addClass("studentcounts").attr("id", "studentcounts_" + section.id).html(section.num_students_attending.toString() + "/" + section.num_students_checked_in + "/" + section.num_students_enrolled + "/" + section.capacity));
```

Comment:
TODO: Only show the class attendance number if the class attendance module is enabled.